packages/framework/src/constants/workflow-options.constants.ts

```diff
@@ -0,0 +1,8 @@
+export enum WorkflowOptionsChannelEnum {
+  EMAIL = 'email',
+  SMS = 'sms',
+  PUSH = 'push',
+  CHAT = 'chat',
+  /** Differs from ChannelTypeEnum in capitalization / snake_case */
+  IN_APP = 'inApp',
+}
```

Comment:
Nice addition 👏

Whilst we're adding this human-friendly enum, it will be great to also update the Step types to use the same enum for the key properties [here](https://github.com/novuhq/novu/blob/next/packages/framework/src/types/step.types.ts#L236). I suggest we make this change in this PR to create a concrete point of reference for when to use this enum.

Lastly, to reflect that this enum is used across both steps and options, I suggest we rename the following:
* filename: `workflow-options.constants.ts` -> `workflow.constants.ts`
* enum: `WorkflowOptionsChannelEnum` -> `WorkflowChannelEnum`